src/merchantCard/log.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace StarkBank\MerchantCard;
+use StarkCore\Utils\API;
+use StarkBank\Utils\Rest;
+use StarkCore\Utils\Checks;
+use StarkCore\Utils\Resource;
+use StarkCore\Utils\StarkDate;
+use StarkBank\MerchantCard;
+
+
+class Log extends Resource
+{
+
+    public $created;
+    public $type;
+    public $errors;
+    public $card;
+
+    /**
+    # Log object
```

Comment:
Faz a mesma coisa nos outros lugares
```suggestion
    # MerchantCard\Log object
```